src/mraa.c

```diff
@@ -116,6 +123,31 @@ imraa_init()
            mraa_get_version(), (proc_user != NULL) ? proc_user->pw_name : "<unknown>", proc_euid);
 
     mraa_platform_t platform_type;
+
+    // Check to see if the enviroment variable has been set
+    env_var = getenv(JSONPLAT_ENV_VAR);
```

Comment:
Not sure about this choice. something prepended with MRAA_ would be safer @arfoll what do you think